circle/docker-release-image.sh

```diff
@@ -48,18 +48,31 @@ RELEASE_SERIES="${TARGET_BRANCH}"
 if [ -n "${VARIANT}" ]; then
   RELEASE_SERIES+="-${VARIANT}"
 fi
-CACHE_TAG=${RELEASE_SERIES}
 
+CACHE_TAG=${RELEASE_SERIES}
```

Comment:
Can we set `CACHE_TAG` always to `${RELEASE_SERIES}-${DISTRO}`?
Then below we append `${RELEASE_SERIES}` to `TAGS_TO_UPDATE` only if is the default distro